source/views/tasks_view.py

```diff
@@ -164,3 +166,14 @@ def get_statuses(self):
         statuses_data = self.status_schema.dump(statuses, many=True).data
         return jsonify({'statuses':statuses_data})
 
+
+    def get_users_per_task(self, id_task):
+        authorization(request.headers.get('Authorization', None))
+        collaborators = TaskOwner.query.filter_by(id_task=id_task)\
+            .filter(TaskOwner.id_task_owner!=TaskOwner.id_user).all()
+        collaborators_list = list()
+        for collaborator in collaborators:
+            task_user = User.query.get(collaborator.id_user)
+            collaborators_list.append(task_user)
+        collaborators_data = self.user_schema.dump(collaborators_list, many=True).data
+        return jsonify({'task_users':collaborators_data})
```

Comment:
Change title from "task_users" to "users", the object itself is User type. It doesn't matter if you're filtering using tasks.